site/config/custom_configuration.md

```diff
@@ -0,0 +1,34 @@
+## Custom Configuration
+
+WDT allows you to create or extend the pre-installed type definitions, model filters, variable injectors, and target environments. Starting with WDT 1.10.0, these additional configuration files can be stored outside the `$WLSDEPLOY_HOME/lib` directory. This allows the files to remain in place if the WDT installation is moved or upgraded to a new version.
+
+To use a separate configuration directory, set the `WDT_CUSTOM_CONFIG` environment variable to the directory to be used for configuration. For example:
+```
+export WDT_CUSTOM_CONFIG=/etc/wdtconfig
+```
+
+The customized configuration files should be named and organized the same way they would be under the `$WLSDEPLOY_HOME/lib` directory. For example:
```

Comment:
We should make it more obvious that the file name is how the CLA selects it by name.

Reply:
Several of these are not specified in CLA. I broke out bullets for each type, and referenced other docs for details. I linked the typedef example here, that goes into more detail about CLS name.